robottelo/ui/base.py

```diff
@@ -282,15 +282,22 @@ def create_a_bookmark(self, name=None, query=None, public=None,
         self.click(locators['bookmark.create'])
 
     def handle_alert(self, really):
-        """
-        Handles any alerts
-        """
+        """Handles any alerts"""
+        alert = self.browser.switch_to_alert()
+        if really:
+            alert.accept()
+        else:
+            alert.dismiss()
+
+    def get_alert_text(self, really=True):
```

Comment:
That's confusing name, i didn't get it by looking through the test. If the method gets the text - then it shouldn't click anything. What you've implemented is basically `handle_alert_and_get_text`.
My 2 cents:
1) `get_alert_text` should return the text and that's it. The test will become intuitive like
```python
text = self.hosts.get_alert_text()
self.assertEqual(text, 'smth')
self.hosts.handle_alert()
```
2) Since `get_alert_text` duplicates `handle_alert` and `handle_alert` doesn't return anything - we can drop `get_alert_text` and extend `handle_alert()` instead to return the text. If specific test needs to verify alert text, it catches the output of `handle_alert`:
```python
text = self.hosts.handle_alert()
self.assertEqual(text, 'smth')
```
Both options are much more intuitive for me than `text = self.hosts.get_alert_text(); self.assertIsNone(self.hosts.search(name))`
Pls share your thoughts.

Reply:
sorry, it can't :))))) anything, literally ANY LINE except alert close will instantly crash the session

Reply:
I thought about extending `handle_alert`. Hmm, it is used in too many places, not sure whether it will work fine for any possible situation.

Reply:
yes, we can rename the method, if you like

Reply:
`return` won't have any effect on tests which don't catch it. The only thing is `alert.text` - if alert has this attribute for sure (even for empty alerts) - then there won't be any issues. Or actually we can insure like that:
```python
message = alert.text if hasattr(alert, 'text') else ''
# handle alert
return message
```